test/remote/payment_gateways_test.rb

```diff
@@ -1,13 +1,12 @@
 require File.expand_path(File.dirname(__FILE__) + '/../test_helper')
-require 'fakeweb'
 
 class PaymentGatewaysTest < ActiveSupport::TestCase
   def setup
     @provider_account = FactoryBot.create(:provider_account)
     @provider_account.billing_strategy = FactoryBot.create(:postpaid_billing)
 
     @buyer_account = FactoryBot.create(:buyer_account, :provider_account => @provider_account)
-    FakeWeb.allow_net_connect = true
+    WebMock.allow_net_connect!
```

Comment:
I know that this is respecting the previous behaviour but we shouldn't make real http requests in the tests 😄

Reply:
I'd even say that we are not using this for this test at all 😕 so I'd try removing it and see if the tests pass 😄